demos/forty/canvas.cpp

```diff
@@ -42,9 +42,9 @@ FortyCanvas::FortyCanvas(wxWindow* parent, const wxPoint& pos, const wxSize& siz
     SetScrollbars(0, 0, 0, 0);
 
 #ifdef __WXGTK__
-    m_font = wxTheFontList->FindOrCreateFont(12, wxROMAN, wxNORMAL, wxNORMAL);
+    m_font = wxTheFontList->FindOrCreateFont(12, wxFONTFAMILY_ROMAN, wxFONTSTYLE_NORMAL, wxFONTWEIGHT_NORMAL);
```

Comment:
This can't be abbreviated currently, unfortunately, but maybe it'd be worth adding `FindOrCreateFont(const wxFontInfo&)` overload? Then we could do the same thing as for the bombs font above.